rwengine/src/script/ScriptModule.hpp

```diff
@@ -175,9 +175,11 @@ class ScriptModule {
     std::map<ScriptFunctionID, ScriptFunctionMeta> functions;
 };
 
+#if !defined(_MSC_VER)
 template <>
 void ScriptModule::bind<void(const ScriptArguments&)>(
     ScriptFunctionID id, int args, void (*function)(const ScriptArguments&));
+#endif
```

Comment:
If this is not necessary for MSVC, why is this necessary in the first place?
Unless there's a compiler bug / extension I'd assume such things to be the same across compilers with the same C++ std.